apps/test-pwmin/src/main.cpp

```diff
@@ -0,0 +1,57 @@
+#include "PwmIn.h"
+#include "mbed.h"
+
+DigitalOut led(LED1);
+
+// Wire the output PWM signal to the PWM input
+// or connect external PWM input to pwmIn pin
+PwmOut pwmOut(PinName::MTR_PWM_TRNTBL);
+PwmIn pwmIn(PinName::ENC_PWM_TRNTBL, 50);
+
+Timer timer;
+Timer printTimer;
+
+int main() {
+  float inverter = +1;
+  auto period    = 2ms;  // Equivalent to 0.5kHz frequency
+  float duty     = 0.1;
+
+  // Specify PWM period
+  pwmOut.period(period.count());
```

Comment:
`period.count()` returns `2`, period parameter is `float seconds`

Reply:
oof lemme double check all the counts

Reply:
caught some more and fixed em